docs/python_docs/python/tutorials/packages/gluon/text/gnmt.rst

```diff
@@ -391,7 +390,7 @@ testing datasets.
 
 
     def write_sentences(sentences, file_path):
-        with io.open(file_path, 'w', encoding='utf-8') as of:
+        with open(file_path, 'w', encoding='utf-8') as of:
```

Comment:
Isn't utf8 by default?

Reply:
No. Default is platform dependent. https://docs.python.org/3/library/functions.html#open